mifosng-android/src/main/java/com/mifos/mifosxdroid/online/surveylist/SurveyListFragment.java

```diff
@@ -80,6 +86,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
 
     @Override
     public void showAllSurvey(final List<Survey> surveys) {
+        if (surveys.size() == 0) {
+            surveySelectText.setText(R.string.no_survey);
+        }
         SurveyListAdapter surveyListAdapter = new SurveyListAdapter(getActivity(), surveys);
```

Comment:
@mayank-kgp if
```
surveys.size() == 0
```

Why are you creating a Survey List Adapter with it ? Don't you think you should put the rest of the code in else block ?